src/screens/OperationDetails/Content.js

```diff
@@ -224,35 +224,41 @@ class Content extends PureComponent<Props, State> {
             })}
           </LText>
         </View>
-        <View style={styles.section}>
-          <LText style={styles.sectionTitle}>
-            <Trans i18nKey="operationDetails.fees" />
-          </LText>
-          {operation.fee ? (
-            <View style={styles.feeValueContainer}>
+        {operation.type !== "IN" ? (
```

Comment:
prefer `==="OUT"`

"NONE" case is not rendered, but if it was, we would not want fees neither, it's really just the OUT

Reply:
Done !